src/main/java/org/folio/circulation/domain/policy/FixedDueDateSchedules.java

```diff
@@ -33,18 +32,34 @@ static FixedDueDateSchedules from(JsonObject representation) {
     }
   }
 
+  public List<JsonObject> getSchedules() {
+    return schedules;
+  }
+
   public Optional<DateTime> findDueDateFor(DateTime date) {
     return findScheduleFor(date)
       .map(this::getDueDate);
   }
 
+  public Optional<DateTime> findEarliestDueDateFor(DateTime date) {
```

Comment:
I don't really understand what this method is doing.

What is it intended to do?

Can you help me understand what scenario it matches up with in the story?